<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Depozit</class>
 <widget class="QWidget" name="Depozit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Depozit</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_16">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Сумма вклада</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="depositAmount">
           <property name="maximum">
            <double>10000000000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Срок размещения</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="depositPeriod">
           <property name="maximum">
            <number>1000000000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Процентная ставка</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="interestRate">
           <property name="maximum">
            <double>10000000000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Налоговая ставка</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="taxRate">
           <property name="maximum">
            <double>10000000000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Периодичность выплат</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="paymentFrequency">
           <property name="maximum">
            <number>100000000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="compoundInterest">
         <property name="text">
          <string>Капитализация</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Список пополнений</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Время транз.</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox">
                 <property name="maximum">
                  <number>100</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Сумма транз.</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBox">
                 <property name="maximum">
                  <double>10000000000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Внести в таблицу</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Список снятий</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>Время транз.</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_2">
                 <property name="maximum">
                  <number>100</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>Сумма транз.</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
                 <property name="maximum">
                  <double>10000000000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>Внести в таблицу</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Начисленные проценты</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="interestAccrued">
           <property name="maximum">
            <double>10000000000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Сумма налога</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="totalTax">
           <property name="maximum">
            <double>10000000000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_15">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Сумма вклада к концу срока</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="finalDepositAmount">
           <property name="maximum">
            <double>10000000000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Рассчитать</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
